runtime/utils.go

```diff
@@ -76,3 +79,25 @@ func PreInitMaster0(sshClient ssh.Interface, remoteHostIP string) error {
 	}
 	return nil
 }
+
+func GetKubectlAndKubeconfig(ssh ssh.Interface, host string) error {
+	// fetch the cluster kubeconfig, and add /etc/hosts "EIP apiserver.cluster.local" so we can get the current cluster status later
+	err := ssh.Fetch(host, path.Join(common.DefaultKubeConfigDir(), "config"), common.KubeAdminConf)
+	if err != nil {
+		return errors.Wrap(err, "failed to copy kubeconfig")
+	}
+	err = utils.AppendFile(common.EtcHosts, fmt.Sprintf("%s %s", host, common.APIServerDomain))
```

Comment:
need check etc host `common.APIServerDomain` is exist,some node this host config host is  localhost or vip

Reply:
AppendFile already check the content is already exist, but not check the APIServerDomain, I will fix it in other PR